### PR TITLE
Modify the second objective of the episode 03-matplotlib

### PR DESCRIPTION
**Pull request by TRUONG3**
I think it's a bit better to change the second objective of this episode from "Group several graphs ..." to "Plot a group several graphs ..."

### CODE REVIEW

**Comment by ldko** [comment]
Hi @TRUONG3 ,
Thank you for taking the time to suggest a change to the objective.  To help us know what you would like to see clarified, could you please share your reasoning for changing the text?

**Comment by TRUONG3** [comment]
Hi, I think the second objective should be changed in this way to better parallel with the 1st objective, which is firstly doing the coding to plot a single graph and then secondly to plot a group of graphs. The coding is indeed for plotting as well, not just grouping the graphs.

File: _episodes/03-matplotlib.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ questions:
 - "How can I group several plots together?"
 objectives:
 - "Plot simple graphs from data."
-- "Group several graphs in a single figure."
+- "Plot a group several graphs in a single figure."
```
Comment (ldko) [inline _episodes/03-matplotlib.md:10]: Perhaps we could cut this down to:
```suggestion
- "Plot multiple graphs in a single figure."
```

**Review by ldko — COMMENTED** [review]
Hi @TRUONG3 , Thanks for giving us the reasoning behind your change. What you said makes sense to me about emphasizing plotting over grouping only.

File: _episodes/03-matplotlib.md (hunk 1 above)
Comment (TRUONG3) [reply _episodes/03-matplotlib.md:10]: yes, better so

**Comment by ldko** [comment]
Thank you for contributing an improvement to the lesson @TRUONG3 !

**Pull request merged**